### ISSUE TITLE
React 15.2.0 warning: Unknown prop `literal` on <p> tag

### ISSUE THREAD

**Issue by tajo** [rexxars/react-markdown#30]
[React 15.2.0 throws a warning if you try to assign custom props to DOM elements](https://gist.github.com/jimfb/d99e0678e9da715ccf6454961ef04d1b). Custom props should be assigned only to React components. 

```
warning.js?8a56:44Warning: Unknown prop `literal` on <p> tag. Remove this prop from the element. For details, see https://fb.me/react-unknown-prop
    in p (created by ReactMarkdown)
    in div (created by ReactMarkdown)
    in ReactMarkdown (created by MY_APP_COMPONENT)
```

This warning appears with any source input.
I just had to fix [the same thing in my library](https://github.com/tajo/react-portal/commit/af6cd34ab46281c62aca579be709d562692c34ba).

**Comment by cythrawll** [comment]
react ticket reference: https://github.com/facebook/react/pull/6800

**Comment by tajo** [comment]
In meantime, I can recommend this alternative https://github.com/vincent-p/react-marked-markdown - it's just a wrapper for https://github.com/chjj/marked - works great.

**Comment by ktalebian** [comment]
I also get 

```
debugger.js:11382 Warning: Unknown props `level`, `literal`, `nodeKey` on <h2> tag. Remove these props from the element. For details, see https://fb.me/react-unknown-prop
    in h2 (created by Heading)
    in Heading (created by ReactMarkdown)
    in div (created by ReactMarkdown)
```

**Comment by rexxars** [comment]
Solved in 2.4.0.
